synapseclient/client.py

```diff
@@ -2818,6 +2818,11 @@ def _downloadWikiAttachment(self, owner, wiki, filename, destination=None):
             destination = os.path.join(destination, filename)
         return self._downloadFile(url, destination)
 
+    def getWikiAttachments(self, wiki):
```

Comment:
Looks good.  We need integration test for this.  If you don't have the bandwidth we can see if we can do it on our side.

P.S.  I like that we are finally getting all of those PEP8 violating spaces cleaned-up :)